docs/snippets/src/main.c

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! [include]
+#include "openvino/c/openvino.h"
+//! [include]
+
+//! [part1]
+ov_core_t* core = NULL; // need to free by ov_core_free(core)
+ov_core_create(&core);
+//! [part1]
+
+{
+//! [part2_1]
+ov_compiled_model_t* compiled_model = NULL; // need to free by ov_compiled_model_free(compiled_model)
+ov_core_compile_model_from_file(core, "model.xml", device_name, 0, &compiled_model);
+//! [part2_1]
+}
+{
+//! [part2_2]
+ov_compiled_model_t* compiled_model = NULL; // need to free by ov_compiled_model_free(compiled_model)
+ov_core_compile_model_from_file(core, "model.onnx", device_name, 0, &compiled_model);
+//! [part2_2]
+}
+{
+//! [part2_3]
+ov_compiled_model_t* compiled_model = NULL; // need to free by ov_compiled_model_free(compiled_model)
+ov_core_compile_model_from_file(core, "model.pdmodel", device_name, 0, &compiled_model);
+//! [part2_3]
+}
+{
+//! [part2_4]
+// Construct a model
+ov_model_t* model = NULL; // need to free by ov_model_free(model)
+ov_core_read_model(core, "model.xml", NULL, &model);
+ov_compiled_model_t* compiled_model = NULL; // need to free by ov_compiled_model_free(compiled_model)
+ov_core_compile_model(core, model, device_name, 0, &compiled_model);
+//! [part2_4]
+}
+
+//! [part3]
+ov_infer_request_t* infer_request = NULL; // need to free by ov_infer_request_free(infer_request)
+ov_compiled_model_create_infer_request(compiled_model, &infer_request);
+//! [part3]
+
+//! [part4]
+ov_tensor_t* tensor = NULL; // need to free by ov_tensor_free(tensor)
```

Comment:
@kblaszczak-intel Why in the documentation we have a "\" before the *?
<img width="919" alt="Screenshot 2022-10-05 at 19 41 11" src="https://user-images.githubusercontent.com/4737252/194103097-89a0738f-8109-42f4-8a69-c5792d1fa56a.png">

Reply:
@kblaszczak-intel Can we remove unnecessary `\` before the `*`?